.github/workflows/push_image_to_dockerhub.yml

```diff
@@ -0,0 +1,54 @@
+# Build the new Docker image on every commit to master/develop branch and on every new tag.
+# No caching is involved for the image build. The new image is then pushed to the Docker Hub.
+
+name: build-and-push-to-dockerhub
+
+on:
+  push:
+    branches:
```

Comment:
Listing these two branches here should not be necessary, but I think it will also not hurt.

If we also want the head of the `support/*` branches to be pushed to dockerhub, we would need to specify that here (but this is not necessary).
The tags should be enough.